spyder/plugins/application/container.py

```diff
@@ -302,31 +302,40 @@ def _check_updates_ready(self):
                 header = _("<b>Spyder {} is available!</b> "
                            "<i>(you have {})</i><br><br>").format(
                     latest_release, __version__)
-                content = ""
+                download_msg = _(
+                    "Would you like to automatically download and "
+                    "install it?<br><br>"
+                )
```

Comment:
Still not 100% happy with it, but here's a shot at a message to add if Spyder is not currently running from  a standalone installer.

```suggestion
                )
                installer_message = _(
                    "Clicking Yes will download and install the latest "
                    "version of Spyder using our standalone installers, "
                    "leaving your existing installation untouched. "
                    "Otherwise, follow the instructions below to update your "
                    " existing Spyder installation.<br><br>"
                )
```

Reply:
You defined `installer_message` but did not use it anywhere. Also, "leaving your existing installation untouched" does not apply to existing installers.

Reply:
Oops, I forgot to rename it to `not_installer_message`, to reflect that it only fires if the user is running Spyder _not_ from an installer (from a platform that supports them). But I'm sure we can do a better job with the naming/implementation, as I was tired, stressed and in a rush.